### ISSUE TITLE
BUG Entities not visible in cesium 1.42 when HTML page loaded as a file://

### ISSUE THREAD

**Issue by crubier** [AnalyticalGraphicsInc/cesium#6282]
### Description

This bug appeared in cesium 1.42, it did not exist before.

We are using cesium inside a web view packaged in an iOS and Android app. So we are loading the html page using the `file://` protocol, not the typical `http://` or `https://` protocols.

Using this protocol, we are able to have a running instance of cesium, with any version of cesium, including 1.42. So far, so good.

**However with cesium 1.42, the entities that are added to the viewer are not visible at all in the viewer !**

Entities are added correctly to cesium, we are able to `getEntityById` and so on, they just don't show up in the viewer.

### Example

To see the bug in action on desktop:

  - `git clone https://github.com/crubier/cesium-webpack-example.git`
  - `npm install`
  - `npm run build` to build the application using webpack
  - Launch chrome with the `--allow-file-access-from-files` flag to allow loading local files, as is done on web views on android or iOS. ( for example by running `/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --allow-file-access-from-files`  on MacOS)
  - Open `dist/index.html` with Chrome
  - Enjoy the bug with `cesium@1.42.0`, and the absence of bug with any previous version...

The repo with the example can be found here: https://github.com/crubier/cesium-webpack-example . It is just a small fork of the standard `cesium-webpack-example`, with these modifications:

  - display a simple sphere at latitude=0 and longitude=0 instead of fireworks
  - change `CESIUM_BASE_PATH` to `./` in order to comply with file loading

**Comment by ggetz** [comment]
Hi @crubier. This sounds to me like the `Worker` files are not being properly loaded. They are what create the geometry to be rendered for each of the Entities. I suspect you may need another configuration for `CESIUM_BASE_URL`, the explanation for which is in #3935.

**Comment by crubier** [comment]
Hi @ggetz , I am sorry but this issue definitely needs to be reopened.

Your analysis is wrong for the following reasons:

- The problem **did NOT exist** with versions prior to 1.42 , with all these versions **the `CESIUM_BASE_URL` that is configured in the example repo is exactly the right one**, 100%. Everything works wonderfully without a single warning in the console. But **suddenly when we switch to version 1.42, this stops working**, without changing anything in the code. 
- The Workers file **are properly loaded, EVEN with version 1.42** . The network tab of the developer tools show that they are loaded with no problem. Just try it yourself, or I can provide a screenshot if you don't believe me.
- Being a long time heavy user of Cesium in fancy configurations (Mobile apps, Web with webpack more than a year ago, etc...) I bumped **many times** into the "wrong `CESIUM_BASE_URL`" errors, and I believe that I know how to recognize them when they happen. I understand these errors might pop up often in Cesium Issues on Github, hence the quick dismissal from your part. But believe me, this time the case is not that trivial, this looks like a different bug caused by changes in Cesium 1.42.

A quick overview of the changelog for cesium gives me the impression that the bug is probably caused by a change in the way URLs are treated internally in Cesium. In particular this sentence of the changelog caught my attention: "Major refactor of URL handling". I might be wrong but I suspect the root of the problem has something to do with that.

TL;DR: Please reopen

Thank you :-)

**Comment by hpinkos** [comment]
Hi @crubier!  We actually fixed a bug with `CESIUM_BASE_URL` earlier this month.  The pull request can be found here: https://github.com/AnalyticalGraphicsInc/cesium/pull/6231

This fix will be included in the Cesium 1.43 release available this Thrusday, March 1st.

If you are still experiencing the problem in that release, please let us know and we will re-open this issue.  Thanks!

**Comment by crubier** [comment]
Ahh ok nice ! 

Thank you very much, much appreciated ! 

Now I agree with closing this :-)
